src/components/Nav.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { Outlet, Link, NavLink } from 'react-router-dom';
+
+export default function Nav() {
+  return (
+    <>
+      {' '}
+      <nav>
+        <h1>
+          <Link to="/">Bookstore CMS</Link>
+        </h1>
+        <ul>
+          <li>
+            <NavLink
+              style={({ isActive }) => (isActive ? { color: '#f5913e' } : {})}
```

Comment:
- [ ] [OPTIONAL] I would like to recommend you to not use inline css Because by using inline CSS, you mix style logic with your component logic, making your code less modular and harder to maintain. It's better to separate style from content by using external CSS files or inline styling approaches like CSS modules. Please fix it 😄